src/explore-education-statistics-common/src/services/util/combineMeasuresWithDuplicateLocationCodes.ts

```diff
@@ -56,28 +56,25 @@ export default function combineMeasuresWithDuplicateLocationCodes(
 
   // Otherwise, locate the potentially affected rows of data in order to merge them.  Find any rows belonging to a
   // deduplicated Location.
-  const [deduplicatedLocationsResults, unaffectedResults] = partition(
-    results,
-    result => {
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      const { code } = result.location![result.geographicLevel];
-      return deduplicatedLocations.find(
-        dedupedLocations =>
-          dedupedLocations.level === result.geographicLevel &&
-          dedupedLocations.value === code,
-      );
-    },
-  );
+  const [affectedResults, unaffectedResults] = partition(results, result => {
```

Comment:
Just wondering why we've renamed the first variable in the array?

Reply:
Reverted this name change now, as I want to impact as little as possible of this code!

I think this code will go after the snapshot migration.